Code/GeorgiaLibrarySystem-/GtlWebsite/Connected Services/ServiceReference1/Reference.cs

```diff
@@ -0,0 +1,137 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace GtlWebsite.ServiceReference1 {
```

Comment:
ServiceReference1 is bad name, lets make better name for reference (later _might_ be hard to change unlike class names)